openedx/features/course_experience/tests/views/test_course_home.py

```diff
@@ -251,6 +252,35 @@ def test_home_page(self, user_type, expected_message):
         if expected_message:
             self.assertContains(response, expected_message)
 
+    @ddt.data(
+        [False, 'private', 'to see course content'],
+        [False, 'preview', 'to see course content'],
+        [False, 'public', 'to see course content'],
+        [True, 'private', 'to see course content'],
```

Comment:
Is this message not going to be visible in all cases?

Reply:
Actually, this was meant for the *You must be enrolled in the course to see course content*, I just shortened it for readability.

But explicit is better, so I'll adjust that.